fedelemflowlist/input/ChemicalsFlowables.csv

```diff
@@ -4635,3 +4635,20 @@ cis-4-Methyl-2-pentene,691-38-3,C6H12 ,"2-pentene, 4-methyl-, (z)-;2-Pentene, 4
 "2-Hexene, 4-Methyl-, (E)-",3683-22-5,C7H14 ,,kg,https://comptox.epa.gov/dashboard/dsstoxdb/results?utf8=%E2%9C%93&search=DTXSID20875696,1
 "Naphtha, petroleum, hydrotreated light",64742-49-0,,"Naphtha, petroleum, hydrotreated light;Naphtha (petroleum), hydrotreated light",kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=15212,1
 Dibromodifluoroethane,31392-96-8,C2H2Br2F2 ,"Ethane, dibromodifluoro-",kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=80374,1
+Ytterbium,7440-64-4,Yb,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36817,1
+Yttrium,7440-65-5,Y,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36816,1
+Zirconium oxide,1314-23-4,ZrO2,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=42863,1
+Thulium,7440-30-4,Tm,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36980,1
+Argon,7440-37-1,Ar,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=36974,1
+Ethyl cellulose,9004-57-3,,kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=37459,1
+Bis(N-Cyclohexyldiazeniumdioxy)copper,312600-89-8,C12H22CuN4O4,"Copper, bis[N-(hydroxy-.kappa.O)-N-(nitroso-.kappa.O)cyclohexanaminato]-",kg,https://ofmpub.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=76843,1
```

Comment:
I'm still getting errors from the extra comma, it's confusing the parsing library. Could we use a semicolon instead? Are we doing this anywhere else where it works as expected?